src/JustSaying/AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -80,11 +80,19 @@ public async Task PublishAsync(Message message, PublishMetadata metadata, Cancel
                 }
             }
 
-            _logger.LogInformation(
-                "Published message: '{SnsSubject}' with content {SnsMessage} and request Id '{SnsRequestId}'",
-                request.Subject,
-                request.Message,
-                response?.ResponseMetadata?.RequestId);
+
+            using (_logger.BeginScope(new[]
+            {
+                new KeyValuePair<string, object>("AwsRequestId", response?.MessageId)
+            }))
+            {
+                _logger.LogInformation(
+                    "Published message {MessageId} of type {MessageType} to {Type} '{Target}'.",
```

Comment:
Yeah good point - perhaps `QueueOrTopic` might make more sense instead of Type. 

For Target, I'm torn between unifying the property names and the fact that we use Arn/QueueUrl depending on if its a Topic/Queue publish.

`PublishDestination` perhaps?

Reply:
Yeah "PublishDestination` seems like a good compromise to me (or `MessageDestination` to keep `Message` as a common prefix).

Reply:
I'll also offer `DestinationType` and `ResourceType`

Reply:
> I'll also offer `DestinationType` and `ResourceType`

Are these both suggestions to replace `QueueOrTopic`? I like `DestinationType` as it matches the `PublishDestination` naming.